samigo/samigo-app/src/webapp/jsf/widget/timerBar/timerbar.js

```diff
@@ -315,11 +315,10 @@
                   elapsedTime = data[1];
                   lastAid = data[2];
                   if (totalTime === elapsedTime) {
-                      $("[id$=\\:save]")[0].click();
```

Comment:
question for @dnhutchins : why is this code starting at line 317 so different from the block of code at line 296?